src/server.ts

```diff
@@ -51,12 +70,12 @@ export async function createServer(
         }
     }
 
-    const server: PluginsServer = {
+    const server: Omit<PluginsServer, 'eventsProcessor'> = {
```

Comment:
This feels hacky. Why not make it optional on the type?

Reply:
It's just omitted for initialization here, but we do want it required on the type.